docs/installation.rst

```diff
@@ -149,9 +149,11 @@ FMRIPREP uses FreeSurfer tools, which require a license to run.
 To obtain a FreeSurfer license, simply register for free at
 https://surfer.nmr.mgh.harvard.edu/registration.html.
 
-When using manually-prepared environments, FreeSurfer will search for a license key
-file first using the ``$FS_LICENSE`` environment variable and then in the default
-path to the license key file (``$FREESURFER_HOME/license.txt``).
+When using manually-prepared environments or singularity, FreeSurfer will search 
+for a license key file first using the ``$FS_LICENSE`` environment variable and then 
+in the default path to the license key file (``$FREESURFER_HOME/license.txt``). If you want
+to use ``$FS_LICENSE`` to point to your license file with singularity remember not to 
+use the ``--clearenv`` flag.
```

Comment:
Maybe what we should suggest is "If using the ``--clearenv`` flag and ``$FS_LICENSE`` is set, use ``--fs-license-file $FS_LICENSE`` to pass the license file location to fMRIPrep."

Reply:
Agreed, and please start the sentence with a newline :+1: